Made it through Foundation and Dispatch successfully. 

XCTest failed a single test on Windows:
```
FAIL: SwiftXCTestFunctionalTests :: Asynchronous/Expectations/main.swift (25 of 26)
******************** TEST 'SwiftXCTestFunctionalTests :: Asynchronous/Expectations/main.swift' FAILED ********************
Script:
--
: 'RUN: at line 1';   T:/1/bin/swiftc.exe -swift-version 5 -vfsoverlay T:/2/stdlib/windows-vfs-overlay.yaml -L T:/5 -I T:/5 -I T:/5\swift -module-cache-path T:/5\XCTest.dir\ModuleCache -Xlinker -nodefaultlib:libcmt -L T:\4 -L T:\4\Foundation -L T:\4\Sources\Foundation -L T:\4\Sources\FoundationNetworking -L T:\4\Sources\FoundationXML -L T:\4\lib -I T:\4 -I T:\4\swift -Xcc -F -Xcc T:\4 -Xcc -fblocks -I C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift-corelibs-libdispatch -I T:\3/src/swift -I T:\3/src/swift\swift -L T:\3 -L T:\3\src -L T:\3\src\BlocksRuntime -L T:\3\src\swift -vfsoverlay T:\3\dispatch-vfs-overlay.yaml C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift-corelibs-xctest\Tests\Functional\Asynchronous\Expectations\main.swift -o C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift-corelibs-xctest\Tests\Functional\Asynchronous\Expectations\Output/Asynchronous
: 'RUN: at line 2';   C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift-corelibs-xctest\Tests\Functional\Asynchronous\Expectations\Output/Asynchronous > C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift-corelibs-xctest\Tests\Functional\Asynchronous\Expectations\Output\main.swift.tmp || true
: 'RUN: at line 3';   'C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python39_64\python.exe' C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift-corelibs-xctest\Tests\Functional\xctest_checker\xctest_checker.py C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift-corelibs-xctest\Tests\Functional\Asynchronous\Expectations\Output\main.swift.tmp C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift-corelibs-xctest\Tests\Functional\Asynchronous\Expectations\main.swift
--
Exit Code: 1276

Command Output (stdout):
--
$ ":" "RUN: at line 1"
$ "T:/1/bin/swiftc.exe" "-swift-version" "5" "-vfsoverlay" "T:/2/stdlib/windows-vfs-overlay.yaml" "-L" "T:/5" "-I" "T:/5" "-I" "T:/5\swift" "-module-cache-path" "T:/5\XCTest.dir\ModuleCache" "-Xlinker" "-nodefaultlib:libcmt" "-L" "T:\4" "-L" "T:\4\Foundation" "-L" "T:\4\Sources\Foundation" "-L" "T:\4\Sources\FoundationNetworking" "-L" "T:\4\Sources\FoundationXML" "-L" "T:\4\lib" "-I" "T:\4" "-I" "T:\4\swift" "-Xcc" "-F" "-Xcc" "T:\4" "-Xcc" "-fblocks" "-I" "C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift-corelibs-libdispatch" "-I" "T:\3/src/swift" "-I" "T:\3/src/swift\swift" "-L" "T:\3" "-L" "T:\3\src" "-L" "T:\3\src\BlocksRuntime" "-L" "T:\3\src\swift" "-vfsoverlay" "T:\3\dispatch-vfs-overlay.yaml" "C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift-corelibs-xctest\Tests\Functional\Asynchronous\Expectations\main.swift" "-o" "C:\Users\swift-ci\jenkins\workspace\swift-PR-windows\swift-corelibs-xctest\Tests\Functional\Asynchronous\Expectations\Output/Asynchronous"
# command stderr:
main-07eda6.o : fatal error LNK1276: invalid directive 'CoreFoundation' found; does not start with '/'
clang: error: linker command failed with exit code 1276 (use -v to see invocation)
<unknown>:0: error: link command failed with exit code 1276 (use -v to see invocation)

error: command failed with exit status: 0x4fc

--

********************
Testing:  0.. 10.. 20.. 30.. 40.. 50.. 60.. 70.. 80.. 90.. 
********************
Failed Tests (1):
  SwiftXCTestFunctionalTests :: Asynchronous/Expectations/main.swift
```

--- a/test/Driver/Dependencies/only-skip-once.swift
+++ b/test/Driver/Dependencies/only-skip-once.swift
@@ -1,4 +1,4 @@
-// XFAIL: OS=linux-gnu, OS=openbsd, OS=linux-android, OS=linux-androideabi
+// XFAIL: OS=linux-gnu, OS=openbsd, OS=windows-msvc, OS=linux-android, OS=linux-androideabi
 
 // RUN: %empty-directory(%t)
 // RUN: cp -r %S/Inputs/only-skip-once/* %t

